### DISCUSSION TITLE
深度IDE有考虑过出跨平台版本嘛？比如说Windows版？

### DISCUSSION THREAD

**Discussion by pffang** [linuxdeepin/developer-center#5430] [Features Request & Ideas | 特性请求 & 头脑风暴]
虽然我觉得vscode可能算是基于Electron开发的性能最好的产品了，但依然摆脱不了Electron固有的大内存占用问题。
我一直期盼有个Native的（C/C++/Rust）、和vscode易用性相仿的轻量IDE。
深度这个看起来就很有希望的样子
有考虑过提供Windows支持吗？

**Comment by deepin-mozart** [comment]
其它系统的支持目前正在讨论，短期内还没有明确的规划，后续有确定消息会告知大家

**Comment by BLumia** [comment]
GitHub Discussion 不能自动展示关联过来的帖子，手动贴一下：

- https://github.com/linuxdeepin/developer-center/discussions/8564
- https://github.com/linuxdeepin/developer-center/discussions/8365
- https://github.com/linuxdeepin/developer-center/discussions/6630
- https://github.com/linuxdeepin/developer-center/discussions/5445
